gradle/groovyProject.gradle

```diff
@@ -46,7 +46,7 @@ testTasks.all { task ->
 jarTasks.all { jar ->
     jar.manifest.mainAttributes(
         (Attributes.Name.IMPLEMENTATION_TITLE.toString()): 'Gradle',
-        (Attributes.Name.IMPLEMENTATION_VERSION.toString()): version,
+        (Attributes.Name.IMPLEMENTATION_VERSION.toString()): versionBase
```

Comment:
Should we then change title to `SPECIFICATION_TITLE`, too? We could just leave then out. Do we have any uses for those manifest entries?